### DISCUSSION TITLE
What is the "new_ComfyUI_windows_portable_nvidia_cu121_or_cpu.7z"

### DISCUSSION THREAD

**Discussion by Robson1970** [comfyanonymous/ComfyUI#1774] [General]
As the title says,  https://github.com/comfyanonymous/ComfyUI/releases/tag/latest  New ComfyUI  ?

**Comment by comfyanonymous** [comment]
That's just the name of the standalone package when it gets uploaded by the github action that creates them:
https://github.com/comfyanonymous/ComfyUI/blob/master/.github/workflows/windows_release_package.yml#L97
